app/src/main/java/com/mediinbusan/app/feature/hospitalsearchlist/HospitalSearchListScreen.kt

```
@@ -667,6 +683,9 @@ private fun EmptySearchBanner(onReset: () -> Unit, modifier: Modifier = Modifier
     }
 }
 
+private fun Double.toDistanceLabel(): String =
+    if (this < 1000.0) "${toInt()}m" else String.format("%.1fkm", this / 1000.0)
```

detekt (1.23.8)

[warning] String.format("%.1fkm", this / 1000.0) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)